va-gov/pages/education/gi-bill/post-9-11/ch-33-benefit.md

```diff
@@ -1,20 +1,23 @@
 ---
-title: Check Benefit
-entryname: post-911-gib-status
-layout: page-react.html
-includeBreadcrumbs: false
-in_maintenance: false
-maintenance_line1: We’re sorry. Our system is temporarily down while we fix a few things. Please try again later.
+layout: page-react-sidebar.html
+title: Check Your Post-9/11 GI Bill Benefits
+display_title: Check Post-9/11 GI Bill Benefits
+description:
```

Comment:
Is this required?

Reply:
Nope, just an extra from the old front matter. Will remove!